### DISCUSSION TITLE
[求助]为什么我换不了头像啊

### DISCUSSION THREAD

**Discussion by Elysia-Ego-093** [share121/inter-knot#249] [其他]
![Screenshot_2024-08-09-23-06-25-610_mark via](https://github.com/user-attachments/assets/e9f581d6-9224-4ae1-9436-7a7098ebb7b8)
如图，想要换头像但是弹出来了这句话，有没有大佬能告诉我是怎么回事啊，谢谢。

**Comment by sabrin1145** [comment]
对不起做不到
![微信图片_20240809232531](https://github.com/user-attachments/assets/78bd88e4-4f71-4160-8d5f-556ad07cdeef)

**Comment by chuhuli** [comment]
嗯呢，嗯呢呢（有格式要求吧，试试别的格式的）

**Comment by Luckyxinchen** [comment]
可能是照片太大了

**Comment by Elysia-Ego-093** [comment]
真的不行啊，换了几十张爱莉的图片都不能通过😭

**Comment by xxtg666** [comment]
试试用电脑换，应该是你那边网络或者浏览器的问题，github不会审核这个

**Comment by wmwcrn** [comment]
要不你试试在电脑端上面改头像

**Comment by lappslover** [comment]
可能是格式问题，我最开始也是换了张图就可以了

**Comment by chuhuli** [comment]
电脑上报错很少，方便了可以试试电脑哦

**Comment by themagicmoon** [comment]
网络问题，谷歌翻出来是说处理不了该图片

**Comment by SteveXXXXiao** [comment]
电脑上问题会少很多，简单来说应该是手机浏览器处理不了这种格式的图片，还要注意的就是图片不能大于1M，多试试应该可以试出来

**Comment by Kamisato100** [comment]
可能是你图片太大了，用PS裁剪压低体积看看
